.github/workflows/check-pr-to-release.yml

```diff
@@ -38,3 +38,30 @@ jobs:
                     echo "Base branch has commits since merge base"
                     exit 1
                   fi
+
+    check_source_branch:
+        name: Check source branch
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Assert source branch is RC branch
+              if: ${{ !startsWith(github.head_ref, 'rc/') }}
+              run: exit 1
+
+    check_release_version:
+        name: Check release version
+        needs: check_source_branch
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Parse version
+              id: version
+              uses: ./.github/parse-version
+
+            - name: Assert release version is not pre-release
+              if: endsWith(steps.version.outputs.releaseVersion, '-pre')
+              run: exit 1
+
+            - name: Assert release version matches RC branch name
+              if: format('rc/{0}', steps.version.outputs.release) != github.head_ref
```

Comment:
maybe i'm missing something, but what is `release`? I don't see any reference to it in the `parse-versions.yml` file?

Maybe i'm supposed to use `releaseShort` instead?

Reply:
using my intuition and adjusted via 685dff8bc60946521c3454f375dfa3c0a685c152